include/boost/geometry/extensions/gis/io/wkb/detail/writer.hpp

```diff
@@ -178,14 +180,16 @@ namespace detail { namespace wkb
 
             interior_rings_type interior_rings = boost::geometry::interior_rings(polygon);
```

Comment:
Great! Thanks for making the change.
`interior_return_type` is indeed better and should avoid the problem I had in mind.